home.html

```diff
@@ -0,0 +1,183 @@
+            <a href="" class="desktopUl-li upperCase">about</a>
+            <a href="" class="desktopUl-li upperCase">sponsors</a>
+            <a href="" class="desktopUl-li upperCase">Persons</a>
+            <a href="" class="desktopUl-li upperCase">contact us</a>
+        </div>
+    </div>
+    </header>
+    <main id="main">
+        <section class="mainInfos">
+            <h1 class="title upperCase">"Hello,Date Lovers"</h1>
+            <h2 class="title-2 upperCase">Dates Festival 2021
+               Mauritania Tagant.</h2>
+               <p class="mainPara">
+                Dates are the fruit of the date palm tree, which is grown in tropical regions of the world.
+                Dates are healthy and delicious and have become very popular recently.
+               </p>
+               <div class="mainDate">
+                <h3 class="eventDate">2021.12.23(FRI) ~ 25(SUN)</h3>
+                <h4 class="eventPlace">@ National Dates Festival of Mauritania, Tagant.</h4>
+            </div>
+        </section>
+        <section class="mainProgram">
+            <h5 class="programTitle capitalize">main program</h5>
+            <div class="separator"></div>
+            <ul class="programContainer">
+                <li class="programli"><img class="mainProgramImg" src="images/camelsImage.jpeg" alt="camels">
+                    <h1 class="mPImgtitle">Camels</h1>
+                    <p class="mPImgPara">Enjoy riding camels in the desert and 
+                        drinking camel milk.
+                    </p>
+                </li>
+                <li class="programli"><img class="mainProgramImg" src="images/yellowDates.jpeg" alt="date">
+                    <h2 class="mPImgtitle">Yellow Dates</h2>
+                    <p class="mPImgPara">The real taste of fruit, this is the date befor becoming darker.
+                    </p>
+                </li>
+                <li class="programli"><img class="mainProgramImg" src="images/poetryImage.jpg" alt="poetryImage">
+                    <h2 class="mPImgtitle">Poetry</h2>
+                    <p class="mPImgPara">Listen to the best poets of the country and exclusive new verses.
+                    </p>
+                </li>
+                <li class="programli"><img class="mainProgramImg" src="images/Amnagel.jpeg" alt="Amnagel">
+                    <h2 class="mPImgtitle">Amnagal</h2>
+                    <p class="mPImgPara">Amnagel is the name of the colorful dates,which are also available.
+                    </p>
+                </li>
+                <li class="programli"><img class="mainProgramImg" src="images/blackDates.jpg" alt="">
+                    <h2 class="mPImgtitle">Black Dates</h2>
+                    <p class="mPImgPara">The classic one enjoy the taste of the fresh black dates.
+                    </p>
+                </li>
+            </ul>
+            <div class="programli pL-Button"><h1>Join Tagant Dates Festival 2021</h1></div>
+        </section>
```

Comment:
The items within the _main program_ section of your design do not follow the typography (**font family**, **font-weight**, ...), and do not have appropriate **padding** and **margins** around the items, as well as **spacing** between each item and around the items. Kindly ensure improve upon these:
- [ ] The title _Main Program_  has bigger and bolder text, as can be seen in the screenshot of the original design below.
- [ ] Add appropriate padding on the extreme left and right sides of the contents.
- [ ] Add appropriate **spacing** (margins/ paddings) between and around the items/ contents.
- [ ] The typography(**font sizes**, **font-weight**, **font color**) should closely match the provided template design.
- [ ] Use **icons** instead  of images
- [ ] The bottom white text link has **no background**

**Your program section:**
![image](https://user-images.githubusercontent.com/12745474/133657281-e3779d01-4215-4a89-904f-4f17f3b31e43.png)


**The original program section:**
![image](https://user-images.githubusercontent.com/12745474/132545825-150c3b77-7c05-4a8c-aa95-956269677499.png)

Reply:
![sp](https://user-images.githubusercontent.com/44745855/133760925-c5b6ec5d-726a-43d3-b9f7-2583203b0f37.png)

Compare with the original you can add some margin-bottom on the `join taggants festival` and the Main program title is missing an `underline` please pay attention to details

Reply:
- [ ] Not addressed. Kindly address this
- [ ] [OPTIONAL] According to the design, there should be some margin around the content of each box.
- [ ] [OPTIONAL] The main program section should also have some margin from the date. See below:

Yours | Original
----- | -----
![image](https://user-images.githubusercontent.com/16058470/133787699-6f1cd028-4f4d-4af3-aacc-f59e4b213e5f.png) | ![image](https://user-images.githubusercontent.com/16058470/133787788-fc9a94ca-709d-4226-a74b-4dd72c2ff3cc.png)